apps/Wallet/src/app/api/safe/history/transactions/route.ts

```diff
@@ -3,8 +3,9 @@ import { getSafeService } from "../../util/utils";
 export async function GET(req: Request, res: Response): Promise<Response> {
   const safeSdk = await getSafeService();
 
-  const txs = await safeSdk.getMultisigTransactions(process.env.SAFE_ADDRESS!);
+  const txs = await safeSdk.getModuleTransactions(process.env.SAFE_ADDRESS!);
```

Comment:
if we do this we will lose transactions that arent about sending eth, ie contract transactions like adding and removing users, these should be visible in the tx history. what about using getAllTransactions instead?

Reply:
Discussed offline, won't fix